prow/config.gen.yaml

```diff
@@ -2468,6 +2468,49 @@ presubmits:
       - emptyDir: {}
         name: varlibdocker
 
+  - name: istio-push-images-2.3
+    trigger: (?m)^/push\-images,?($|\s.*)
+    rerun_command: /push-images
+    skip_report: false
+    max_concurrency: 2
+    always_run: false
+    branches:
+    - ^maistra-2.3$
+    decorate: true
+    path_alias: istio.io/istio
+    labels:
+      preset-quay-pusher: "true"
+    spec:
+      containers:
+      - command:
+        - entrypoint
+        - sh
+        - -c
+        - HUB=quay.io/maistra TAG=PR-${PULL_NUMBER} make init maistra-image.push
```

Comment:
Hey, sorry for not noticed this before. Could you change the hub to `quay.io/maistra-dev`?

Strictly speaking `init` is not required, but this is just a nit

Reply:
Thanks! done.